omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -227,6 +231,24 @@
             {max_age, <%= @solr_http_max_age %>},
             {max_connection_duration, <%= @solr_http_max_connection_duration %>},
             {ibrowse_options, <%= @solr_ibrowse_options %>}
+        ]},
+        {rabbitmq_index_management_service, [
```

Comment:
It is a bit of a shame that in the common case (actions & index rabbitmq being the same) that we are going to have 2 pools of HTTP connections.  However, since thy can be different, I think this is cleanest for now.